builder/src/main/java/dev/sashimono/builder/jar/SignatureTask.java

```diff
@@ -0,0 +1,45 @@
+package dev.sashimono.builder.jar;
+
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.parseBoolean;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.function.Function;
+
+import dev.sashimono.builder.util.TaskMap;
+
+public class SignatureTask implements Function<TaskMap, Void> {
+
+    @Override
+    public Void apply(final TaskMap taskMap) {
+        if (parseBoolean(System.getProperty("sign_artifacts", FALSE.toString()))) {
```

Comment:
You can just use `Boolean.getBoolean("sign_artifacts")` here.

Reply:
Damn, I totally forgot about that. It's been a littlle while since I've used system properties. Cheers.